azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/style/ChatCompositeDimensions.kt

```diff
@@ -19,6 +19,7 @@ internal data class ChatCompositeDimensions(
     val messageAvatarRailWidth: Dp = 32.dp,
     val messageReceiptRailWidth: Dp = 20.dp,
     val messageUsernamePaddingEnd: Dp = 8.dp,
+    val messageListMaxWidth: Dp = 560.dp,
```

Comment:
Shauna, do we have way to use ratio? Not sure if this will work on small screen devices like nexus 5

Reply:
Alex Suggested the max width will be 560dp. Not sure we could use ratio to instead if we have bigger screen. Also I tested with 3.2' screen simulator screenshot as above, it works fine.